### DISCUSSION TITLE
Forecast vs Current Weather.

### DISCUSSION THREAD

**Discussion by ankonaskiff17** [weather-gov/api#433] [Q&A]
There is an open source project called Magic Mirror which has a bunch of semi-formatted what they call modules. One of these modules is called Weather (go figure).  The module can be used for current as shown in settings shown below. There are more pieces and parts associated with module. This is all done in javascript.  There are two main parts to a given module.
One part is where all the actual programming and property definitions are created. Second part is called config.js and is what you see below.  This group of configuration settings produces a nice formatted output to a monitor.
Where I am stuck is in what the apiBase and weatherEndpoint are to bring in the daily forecast for say 5 days. I assume it would be reflected in the weatherEndpoint setting but the documentation on your site about the API talks about gridpoints and I've run through your examples and get a forecast but your apiBase is a little  different that what is shown below.
Do you have any thoughts on this problem. I want to make note of fact I AM NOT a programmer. I can change the configuration settings such as true, false, lat, lon, and the like but thats about it.
I'm chasing ultimately the apiBase and weatherEndpoint equivalent that yields a daily forecast

```
config: {
			weatherProvider: "weathergov",
			type: "current",
			location: "Elgin",
			degreeLabel: true,
			showPeriodUpper: true,
			showHumidity: true,
			lat: 34.13178062588381,
			lon: -80.84202412490625,
			apiBase: 'https://api.weather.gov/points/',
			weatherEndpoint: "/forecast/hourly",
				}
```

**Comment by scadergit** [comment]
We cannot support third party software, but perhaps others that use the same library can offer ideas. The "apiBase" property of the library is not an API construct, so it won't be reflected in our documentation. Your example is pulling the hourly forecast, but perhaps you are looking for the period forecast, which is just /forecast. You might want to share what you get so far and what you expected.

**Reply by ankonaskiff17** [reply]
Has the weather API changed within the last couple of weeks? I was using /forecast and /forecast/hourly  in a site I built and /forecast/hourly stopped working a week or so ago.
I can drop it in a blank tab and it outputs the data as does /forecast. I'm trying to troubleshoot problem and am currently baffled

**Comment by ankonaskiff17** [comment]
I had loaded a link based on your api tutorial right before I minimized that application. Opened back up and it worked so all is good.

Attachments are line for current, daily, and how it renders on screen.
